app/code/Magento/Bundle/Test/Mftf/Test/AdminAddBundleItemsTest.xml

```diff
@@ -40,39 +40,47 @@
-            <argument name="product" value="$$simpleProduct0$$"/>
-        </actionGroup>
-        <checkOption selector="{{AdminAddProductsToOptionPanel.firstCheckbox}}" stepKey="selectFirstGridRow"/>
-        <actionGroup ref="FilterProductGridBySkuActionGroup" stepKey="filterBundleProductOptions2">
-            <argument name="product" value="$$simpleProduct1$$"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="conditionallyOpenSectionBundleItems"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="clickAddOption3"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="waitForBundleOptions"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="fillOptionTitle"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="selectInputType"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="clickAddProductsToOption"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="filterBundleProductOptions"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="selectFirstGridRow"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="filterBundleProductOptions2"/>
+        <comment userInput="Comment is added to preserve the step key for backward compatibility" stepKey="selectFirstGridRow2"/>
+        <actionGroup ref="AddBundleOptionWithTwoProductsActionGroup" stepKey="clickAddSelectedBundleProducts">
+            <argument name="x" value="0"/>
+            <argument name="n" value="1"/>
+            <argument name="prodOneSku" value="$$simpleProduct0.sku$$"/>
+            <argument name="prodTwoSku" value="$$simpleProduct1.sku$$"/>
+            <argument name="optionTitle" value="{{BundleProduct.optionTitle1}}"/>
+            <argument name="inputType" value="{{BundleProduct.optionInputType1}}"/>
         </actionGroup>
-        <checkOption selector="{{AdminAddProductsToOptionPanel.firstCheckbox}}" stepKey="selectFirstGridRow2"/>
-        <click selector="{{AdminAddProductsToOptionPanel.addSelectedProducts}}" stepKey="clickAddSelectedBundleProducts"/>
-        <!-- Check that Bundle Options initialized with default quantity -->
+
+        <!-- Check that Bundle Options initialized with proper quantity -->
         <grabValueFrom selector="{{AdminProductFormBundleSection.bundleOptionXProductYQuantity('0', '0')}}" stepKey="grabbedFirstBundleOptionQuantity"/>
         <assertEquals stepKey="assertFirstBundleOptionDefaultQuantity">
-            <expectedResult type="string">1</expectedResult>
+            <expectedResult type="string">50</expectedResult>
```

Comment:
It looks little bit strange that PR that replace mftf test with action groups contain expectation of different value.
Could you please explain why expected result changed from 1 to 50?

Reply:
Hi, @Den4ik!
As you may see I use existing ActionGroup here - AddBundleOptionWithTwoProductsActionGroup. Unfortunately, this ActionGroup doesn't allow us to set specific product qty. We have prefiled value - 50.

Code from ActionGroup - AddBundleOptionWithTwoProductsActionGroup:
`<fillField selector="{{AdminProductFormBundleSection.bundleOptionXProductYQuantity(x, '0')}}" userInput="50" stepKey="fillQuantity1"/>
        <fillField selector="{{AdminProductFormBundleSection.bundleOptionXProductYQuantity(x, '1')}}" userInput="50" stepKey="fillQuantity2"/>`
        
That's why I changed product qty to 50 in the test. As I may see it is not critical for the test.    
If you think it is bad idea, I will try to find some other actions groups to modify the test code.

Reply:
@kate-kyzyma Thanks for the answer. No all clear for me and looks good